File: tier1/csharp/7b7dde7f/Runner.cs
using System.Diagnostics;

namespace App;

public static class Runner
{
    public static void Archive(string name)
    {
        var info = new ProcessStartInfo("tar");
        info.ArgumentList.Add("-cf");
        info.ArgumentList.Add("backup.tar");
        info.ArgumentList.Add(name);
        Process.Start(info);
    }
}

File: tier1/java/48fcb735/Runner.java
package app;

import java.io.InputStream;
import java.util.List;

public class Runner {
    static String handle(String name) throws Exception {
        ProcessBuilder builder = new ProcessBuilder(List.of("tar", "-cf", "backup.tar", name));
        Process process = builder.start();
        InputStream stream = process.getInputStream();
        return new String(stream.readAllBytes());
    }
}
